<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSetting</class>
 <widget class="QWidget" name="WidgetSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>202</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
font: 18px &quot;.Helvetica Neue DeskInterface&quot;;</string>
  </property>
  <widget class="QPushButton" name="button_change">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>110</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(:/images/button/button4.png);
	color: rgb(66, 66, 66);
	font: 18px &quot;.Helvetica Neue DeskInterface&quot;;
	outline: none
}
QPushButton:pressed{
border-image: url(:/images/button/button4_press.png);
}
</string>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>191</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ERROR</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edit_error">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border-image: url(:/images/tools/inputSmall.png);
	color: rgb(255, 255, 255);
	font: 24px &quot;.Helvetica Neue DeskInterface&quot;;
}

</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>20</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>90</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>705</width>
     <height>150</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>LED</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_LED">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border-image: url(:/images/tools/inputSmall.png);
	color: rgb(255, 255, 255);
	font: 24px &quot;.Helvetica Neue DeskInterface&quot;;
}

</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>TVOC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_TVOC">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border-image: url(:/images/tools/inputSmall.png);
	color: rgb(255, 255, 255);
	font: 24px &quot;.Helvetica Neue DeskInterface&quot;;
}

</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>AVCC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_AVCC">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border-image: url(:/images/tools/inputSmall.png);
	color: rgb(255, 255, 255);
	font: 24px &quot;.Helvetica Neue DeskInterface&quot;;
}

</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>WiFi</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_WIFI">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border-image: url(:/images/tools/inputSmall.png);
	color: rgb(255, 255, 255);
	font: 24px &quot;.Helvetica Neue DeskInterface&quot;;
}

</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>RTC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_RTC">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border-image: url(:/images/tools/inputSmall.png);
	color: rgb(255, 255, 255);
	font: 24px &quot;.Helvetica Neue DeskInterface&quot;;
}

</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>IPSOUT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_IPSOUT">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border-image: url(:/images/tools/inputSmall.png);
	color: rgb(255, 255, 255);
	font: 24px &quot;.Helvetica Neue DeskInterface&quot;;
}

</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>DRAM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_DRAM">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border-image: url(:/images/tools/inputSmall.png);
	color: rgb(255, 255, 255);
	font: 24px &quot;.Helvetica Neue DeskInterface&quot;;
}

</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>5V</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_5V">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border-image: url(:/images/tools/inputSmall.png);
	color: rgb(255, 255, 255);
	font: 24px &quot;.Helvetica Neue DeskInterface&quot;;
}

</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
